src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -155,9 +167,20 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
           />
           <EventHeader event={event} />
           <HeaderBottomControls>
-            <StyledButton size="small" onClick={this.toggleSidebar}>
-              {isSidebarVisible ? 'Hide Details' : 'Show Details'}
-            </StyledButton>
+            <ButtonBar gap={1}>
+              <StyledButton size="small" onClick={this.toggleSidebar}>
+                {isSidebarVisible ? t('Hide Details') : t('Show Details')}
+              </StyledButton>
+              {transactionSummaryTarget && (
+                <StyledButton
```

Comment:
I believe this will hide the button on smaller screens, is this the intention?

Reply:
Yeah, we run out of room as transaction names are pretty long. @doralchan should this button be visible on narrow view ports?